ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md

```diff
@@ -264,8 +228,75 @@ port_profiles:
           short_esi: < 0000:0000:0000 | auto >
 ```
 
+### Network Ports
+
+```yaml
+# Network Ports | Optional
```

Comment:
Not sure if we should indicate Optional here - we don't do it for connected-endpoint and they also are

Reply:
added to connected-endpoints